script/Ciel/Day 7/07_01_CIEL07_9.txt

```diff
@@ -686,23 +741,24 @@ It was fear, together with an inexpressible anger.
 [sha:112a08139e9b7a8da9131eab6317bf071386322c]{
 -- Page 690, Offset 16523.
 -- “だというのに―――<こ|・><の|・><体|・><に|・><は|・>、<あ|・><ま|・><り|・><に|・><も|・><時|・><間|・><が|・><な|・><い|・>”
-'So why... %{i}why does this body of mine have so little time?%{/i}'
+'So then why... why does this body of mine have so little %{i}time%{/i}?'
 }
 [sha:ae7fd4443f79965f0e8a4f8d3a506f7e4a8e6015]{
 -- Page 691, Offset 16524.
 -- 世界の広さに比べて<人間|わたし>の生はあまりにも狭すぎる。
-Compared to the expanse of the world, his human life was so minute.
+His lifespan as a human was dwarfed in comparison with the vastness of the world.
```

Comment:
I vastly prefer 'human life' over 'lifespan as a human', in fact I feel like maybe this should be rewritten a bit more.

Reply:
"human lifespan" is also a valid choice